.githooks/post-checkout

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+. ./.githooks/functions.sh
+
+echo "Git Hook: 'post-checkout' is running!"
+
+check_difference "package-lock.json" "npm install"
```

Comment:
package-lock instead of package.json.... interesting

Reply:
Agreed, package-lock.json > package.json.